### DISCUSSION TITLE
why learning rate is set to 0.1 ? in classification in Chapter 2

### DISCUSSION THREAD

**Discussion by hjshi** [mrdbourke/pytorch-deep-learning#1009] [Q&A]
The learning rate in classification in Chapter 2 is set to 0.1
I try to use 0.01, seems it does not work. 
Any explanation why it needs to set to 0.1?

**Comment by LuluW8071** [comment]
In Chapter 2, it’s just finding (classifying) the regional boundary of blobs. So, we don’t need to use higher learning rates (0.01, 0.001, etc.) as model and dataset aren't complex. Setting a lower learning rate helps the model converge faster, while higher learning rates make the model converge slower.

Setting the learning rate depends on the usage of data and the complexity of the model.

In your case, you can use 0.1 with fewer epochs to see results, or set 0.01 with more epochs.
